src/jats/importer/jats-transformations.ts

```diff
@@ -148,8 +148,11 @@ export const createBody = (
 ) => {
   const group = createSectionGroup(bodyType, createElement)
 
-  const paragraphs = doc.querySelectorAll('body > p')
-  group.append(...paragraphs)
+  const elements = body.querySelectorAll(':scope > *:not(sec)')
+  elements.forEach((element) => {
+    removeNodeFromParent(element)
+    group.appendChild(element)
+  })
```

Comment:
Hi @mnatsheh, I've added support for all elements here instead of paragraphs. Is that okay? and if so, should we only include elements that are before the first section? 
I'm also working on a cleanup that would remove the need for this.

Reply:
That's fine. JATS doesn't allow anything (mostly) after `sec` so we don't need to worry about that.

Reply:
BTW, is this something we missed in the non-section support we added recently?